octobot_trading/exchanges/connectors/exchange_simulator.py

```diff
@@ -110,6 +111,16 @@ def get_available_time_frames(self):
                     for time_frame in backtesting_api.get_available_time_frames(next(iter(self.exchange_importers)))]
         return []
 
+    def get_backtesting_data_files(self):
+        return [backtesting_api.get_data_file_path(importer) for importer in self.exchange_importers]
+
+    def get_backtesting_data_file(self, symbol, time_frame):
+        for importer in self.exchange_importers:
+            if symbol in backtesting_api.get_available_symbols(importer) and \
+                    time_frame in backtesting_api.get_available_time_frames(importer):
+                return backtesting_api.get_data_file_path(importer)
+        return None
```

Comment:
Right, it's not really the exchange's job to know this. (I guess) I put it there because it's the exchange that contains its `exchange_importers` and it was easier this way. We could add an octobot-backtesting api to do the same thing with a list of importers and call it directly from the trading api (from which `get_backtesting_data_file` is called)

Reply:
Yes it would be perfect like that